### PR TITLE
HF - Static "website" check fix

### PR DESCRIPTION
**Pull request by martypowell**
Updated logic to use existing ```requiresLocation``` flag, instead of hard-coded text.

In [dev](https://dev.baltimorecountymd.gov/iwant/report.html)

### CODE REVIEW

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js [hunk 1]
```diff
@@ -240,30 +240,33 @@
...
-				if (self.page === 2 && self.category.name === 'Website Issue') {
-					self.latitude = CONSTANTS.locations.courtHouse.latitude;
-					self.longitude = CONSTANTS.locations.courtHouse.longitude;
-					self.page = 3;
-				}
-
-				if (self.page === 2) {
-					$timeout(mapResize, 500);
-				}
+				setupLocation(3);
```
Comment (mxsnyder) [inline js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: Did you mean to leave the constant in here? `setupLocation(self.page)` would probably be better.

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js [hunk 2]
```diff
@@ -240,30 +240,33 @@
...
 
 		self.prevClick = function prevClick() {
 			self.page -= 1;
 
-			if (self.page === 2 && self.category.name === 'Website') {
-				self.latitude = CONSTANTS.locations.courtHouse.latitude;
-				self.longitude = CONSTANTS.locations.courtHouse.longitude;
-				self.page = 1;
-			}
+			setupLocation(1);
```
Comment (mxsnyder) [inline js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: Did you mean to leave the constant in here? `setupLocation(self.page)` would probably be better.

**Comment by mxsnyder** [comment]
Also, I didn't see the categories JSON file in here. Did you mean to skip that?

**Comment by martypowell** [comment]
@mxsnyder the categories.json file already has the requiresLocation flag.

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js (hunk 1 above)
Comment (mxsnyder) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: Nevermind. I'm having a morning.

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js (hunk 2 above)
Comment (mxsnyder) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: Nevermind. I'm having a morning.

File: js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js [hunk 3]
```diff
@@ -240,36 +241,42 @@
...
+			self.latitude = CONSTANTS.locations.courtHouse.latitude;
+			self.longitude = CONSTANTS.locations.courtHouse.longitude;
+			self.page = pageToShow;
 
-			if (self.page === 2 && self.category.name === 'Website') {
-				self.latitude = CONSTANTS.locations.courtHouse.latitude;
-				self.longitude = CONSTANTS.locations.courtHouse.longitude;
-				self.page = 1;
-			}
+			$timeout(mapResize, 500);
```
Comment (mxsnyder) [inline js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: `$timeout(mapResize, 500);` needs to happen when the location page loads. Right now it looks like that only happens if you're skipping the location page.

Comment (martypowell) [reply js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js]: @mxsnyder good catch, done.

**Comment by martypowell** [comment]
resolves #180

**Pull request merged**